django/conf/global_settings.py

```diff
@@ -261,6 +261,8 @@ def gettext_noop(s):
 #    ]
 IGNORABLE_404_URLS = []
 
+# Prefix for default SECRET_KEY (generated by Django)
+SECRET_KEY_INSECURE_PREFIX = 'django-insecure-'
```

Comment:
I didn't mean adding this as a configurable setting as we don't want users to change it.
```suggestion
```

I meant adding it as a constant next to the existing constants related to checks on the secret key: https://github.com/django/django/blob/471d430f21a4cf6a51ecb4ae04a2c740c0771c41/django/core/checks/security/base.py#L12-L13